easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb

```
@@ -0,0 +1,33 @@
+easyblock = 'PythonBundle'
+
+name = 'Sniffles'
+version = '2.0.7'
+
+homepage = 'https://github.com/fritzsedlazeck/Sniffles'
+description = """A fast structural variant caller for long-read sequencing,
+ Sniffles2 accurately detect SVs on germline, somatic and population-level for PacBio and Oxford Nanopore read data."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('Pysam', '0.20.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    (name, version, {
+        'modulename': '%(namelower)s',
+        'sources': [SOURCELOWER_TAR_GZ],
+        'checksums': ['bf532eb0b3aaeaee840100a3bb3339f1d5175ba98adfe3cf19685b0abb0885a1'],
+    }),
+]
+
+sanity_check_commands = [
+    "python -c 'from sniffles import sv'",
+    "sniffles --help",
+]
```

If there's a custom binary, then also define `sanity_check_paths` to overrule the default which only checks for the `site-packages` dir:

```suggestion
sanity_check_paths = {
    'files': ['bin/sniffles'],
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}

sanity_check_commands = [
    "python -c 'from sniffles import sv'",
    "sniffles --help",
]